src/modules/commerce/index.ts

```diff
@@ -1,6 +1,78 @@
 import type { Faker } from '../../faker';
 import { deprecated } from '../../internal/deprecated';
 
+// Source for official prefixes: https://www.isbn-international.org/range_file_generation
+const ISBN_LENGTH_RULES: {
+  [group: string]: Array<[rangeMaximum: number, length: number]>;
+} = {
```

Comment:
If we simplify the type here:

```suggestion
const ISBN_LENGTH_RULES: Record<
  string,
  Array<[rangeMaximum: number, length: number]>
> = {
```